Hippo.Web/Controllers/OrderController.cs

```diff
@@ -72,10 +72,10 @@ public async Task<IActionResult> AdminOrders()
                 return BadRequest("You do not have permission to view this page.");
             }
             var currentUserAccount = await _dbContext.Accounts.SingleOrDefaultAsync(a => a.Cluster.Name == Cluster && a.OwnerId == currentUser.Id);
-            if (currentUserAccount == null)
-            {
-                return Ok(new OrderListModel[0]);
-            }
+            //if (currentUserAccount == null)
+            //{
+            //    return Ok(new OrderListModel[0]);
+            //}
```

Comment:
Should probably just remove this, since currentUserAccount is no longer referenced.